newIDE/app/src/Tutorial/TutorialContext.js

```diff
@@ -44,7 +44,38 @@ export const TutorialStateProvider = ({
           console.info(
             `Loaded ${allTutorials ? allTutorials.length : 0} tutorials.`
           );
-          setTutorials(allTutorials);
+          setTutorials([
```

Comment:
Was this for testing?

Reply:
Oh damn, I tought I did not commit that 😅